mandatory/2-function-creation.js

```diff
@@ -25,7 +35,15 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  typeof arrayOfStrings === "array";
```

Comment:
Again, this line doesn't do anything -- were you trying to validate the input?

Reply:
Agreed.